openedx/adg/lms/user_authn/password_reset.py

```diff
@@ -0,0 +1,49 @@
+"""
+Password reset forms and views.
+"""
+
+from django.urls import reverse
+from django.utils.translation import ugettext_lazy as _
+
+from openedx.core.djangoapps.user_api import accounts
+from openedx.core.djangoapps.user_api.helpers import FormDescription
+
+
+def get_password_reset_form():
+    """
+    Return a description of the password reset form.
+
+    This decouples clients from the API definition:
+    if the API decides to modify the form, clients won't need
+    to be updated.
+
+    See `user_api.helpers.FormDescription` for examples
+    of the JSON-encoded form description.
+
+    Returns:
+        HttpResponse
+
+    """
```

Comment:
Same, can we please change doc string to elaborate, that we are overriding the core method, may be way can add path of the core method as well.

Reply:
And list the things we have overridden.

Reply:
@ahmed-faraz46 bhai what do you mean by listing the overridden things? On wiki? or where?

Reply:
In the doc string.